controllers/insights/index.js

```diff
@@ -78,7 +80,34 @@ router.get('/documents/:slug?', async function (req, res, next) {
     }
 });
 
-router.use('/covid-19-resources/:slug/:child_slug?', flexibleContentPage());
+router.use('/covid-19-resources/:slug/:child_slug?',
+    async function (req, res, next) {
+        const blogposts = await contentApi.getUpdates({
+            locale: req.i18n.getLocale(),
+            type: 'blog',
+        });
+
+        const context = {
+            ...res.locals,
+            ...req.app.locals,
+            ...{
+               blogposts: blogposts.result
+            }
+        };
+
+        nunjucks.render(
```

Comment:
This is pretty gnarly. Not worth blocking on this but would be good to come back to this if we can and see if there's a way of extending off `flexible-content-page` and capturing a block with the sidebar content, rather than having to call out to nunjucks directly.